src/exchange.js

```diff
@@ -635,6 +639,7 @@ class Exchange extends EventEmitter {
     if (!forceRefreshProducts) {
       try {
         formatedProducts = await readProducts(this.id)
+        console.log(formatedProducts)
```

Comment:
comment

Reply:
I have pushed the changes. Thank you for catching this.